google_guest_agent/run/run_test.go

```diff
@@ -0,0 +1,188 @@
+//  Copyright 2023 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package run
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path"
+	"strings"
+	"testing"
+	"time"
+)
+
+// builds a set of data to be used in the tests
+func buildDataContent(t *testing.T) (string, error) {
+	t.Helper()
+	rootDir := path.Join(t.TempDir(), fmt.Sprintf("%d", time.Now().UnixNano()))
+	if err := Quiet("mkdir", "-p", rootDir); err != nil {
+		return rootDir, err
```

Comment:
Since `t` is already passed here we can do `t.Fatalf` here itself instead of returning and checking for error again at callers.
Also, since this is helper method and also used for testing `Quiet()`, I think we should not use it here and rather rely on standard library for that?

Reply:
Yes and Yes. I'll get it done. Thanks.

Reply:
Done. Thanks.